paddle/fluid/operators/mkldnn/caching_tests.cmake

```diff
@@ -1 +1,6 @@
-cc_test(test_mkldnn_caching SRCS mkldnn/test_mkldnn_caching.cc DEPS op_registry elementwise_mul_op elementwise_add_op activation_op softmax_op softmax scope device_context enforce)
+set(TEST_MKLDNN_CACHING_DEPS op_registry elementwise_mul_op elementwise_add_op activation_op softmax_op conv_op im2col vol2col softmax scope device_context enforce)
+if (WITH_GPU OR WITH_ROCM)
+  set(TEST_MKLDNN_CACHING_DEPS ${TEST_MKLDNN_CACHING_DEPS} depthwise_conv)
+endif()
+cc_test(test_mkldnn_caching SRCS mkldnn/test_mkldnn_caching.cc DEPS ${TEST_MKLDNN_CACHING_DEPS})
```

Comment:
Why depthwise_conv needs to be in DEPS while the test did not use it?
If it is really needed, what about always put depthwise_conv always as one of the DEPS. Cause I think Baidu has other hardware too.

Reply:
conv on GPU got depedepndency on depthwise_conv. Without it there is linking error: unresolved DepthwiseConv.... So i had to add it